crates/primitives/src/transaction/signature.rs

```diff
@@ -93,19 +93,19 @@ impl Signature {
     }
 
     /// Output the length of the signature without the length of the RLP header
-    pub(crate) fn payload_len(&self) -> usize {
+    pub fn payload_len(&self) -> usize {
         self.odd_y_parity.length() + self.r.length() + self.s.length()
     }
 
     /// Encode the `odd_y_parity`, `r`, `s` values without a RLP header.
-    pub(crate) fn encode(&self, out: &mut dyn reth_rlp::BufMut) {
+    pub fn encode(&self, out: &mut dyn reth_rlp::BufMut) {
         self.odd_y_parity.encode(out);
         self.r.encode(out);
         self.s.encode(out);
     }
```

Comment:
why does this need to be public now?

this should include a note that this does not encode the header 

should we implement the trait instead?